pkg/controller/controller.go

```diff
@@ -86,20 +87,20 @@ func (c *Controller) Run(stopChannel <-chan struct{}) error {
 	go wait.Until(c.updatePreSharedCertAnnotation, time.Minute, stopChannel)
 
 	glog.Info("Controller waiting for stop signal or error")
-	select{
-		case <-stopChannel:
-			glog.Info("Controller received stop signal")
-			quit(mcertStopChannel, ingressStopChannel)
-		case err := <-errors:
-			runtime.HandleError(err)
-			quit(mcertStopChannel, ingressStopChannel)
+	select {
+	case <-stopChannel: // [review] wait for close
```

Comment:
Sorry, I don't understand this comment. I could do it like that:
for {
 select {
  case _, more := <-stopChannel:
   if more {
    break
   }
  case err := <-errors:
   break
  }
}

Did you mean that?